src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx

```diff
@@ -54,6 +54,7 @@ export const EstimateYourBenefitsSummarySheet = props => (
   <div className="vads-u-padding-bottom--1p5 vads-u-border-top--1px vads-u-border-color--gray-light">
     <button
       aria-expanded={props.expandEybSheet ? 'true' : 'false'}
+      aria-controls={props.showEybSheet && 'eyb-sheet'}
```

Comment:
will naming this `eyb-summary-sheet` cause an issues?

Reply:
That should not cause any issues.

Reply:
That should not cause any issues